src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -263,10 +266,12 @@ private struct InterpolatedStringScanner
             public SyntaxDiagnosticInfo? Error = null;
             private bool EncounteredUnrecoverableError = false;
 
-            public InterpolatedStringScanner(Lexer lexer, bool isVerbatim)
+            public InterpolatedStringScanner(Lexer lexer)
             {
                 _lexer = lexer;
-                _isVerbatim = isVerbatim;
+
+                _isVerbatim = (lexer.TextWindow.PeekChar(0) == '$' && lexer.TextWindow.PeekChar(1) == '@') ||
+                              (lexer.TextWindow.PeekChar(0) == '@' && lexer.TextWindow.PeekChar(1) == '$');
```

Comment:
Too bad we can't do something like `is ['$', '@', ..] or ['@', '$', ..]`, because we probably don't want to pull on the length here, but it would look really cool ;-)

Reply:
yup yup.